src/css/flags/time-slider-above.less

```diff
@@ -358,7 +358,7 @@
     }
 
     /* Move non-linear ad display above the time slider */
-    .jw-plugin:not(.jw-plugin-related) {
+    .jw-plugin {
```

Comment:
Don't think I made this, change, might have been a change that @madikarizma made, or a bad merge. @jw-kaurand this css should not really do anything to related because the overlay should would cover up the time slide, correct?